docs/cis_self_assessment.md

```diff
@@ -735,9 +741,20 @@ Ensure that the admission control plugin SecurityContextDeny is set if PodSecuri
 SecurityContextDeny can be used to provide a layer of security for clusters which do not have PodSecurityPolicies enabled.
 </details>
 
-**Result:** Pass
+**Result:** **Not Scored - Operator Dependent**
+
+**Audit:**
+Run the below command on the master node.
+
+```bash
+/bin/ps -ef | grep kube-apiserver | grep -v grep
+```
+
+Verify that the `--enable-admission-plugins` argument is set to a value that includes `SecurityContextDeny`, if `PodSecurityPolicy` is not included.
 
 **Remediation:**
+By default, RKE2 only sets `NodeRestriction,PodSecurityPolicy` as the parameter to the `--enable-admission-plugins` argument.
+To configure this, follow the Kubernetes documentation and set the desired limits in a configuration file. Then refer to RKE2's documentation to see how to supply additional api server configuration via the kube-apiserver-arg parameter.
```

Comment:
Since the user doesnt need to do anything to get this to pass, we can drop the "to configure" line and replace it with the "no manual...." line